app/helpers/format-config.js

```diff
@@ -1,5 +1,8 @@
 import { safe, formatConfig as formatConfigHelper } from 'travis/utils/helpers';
 
-export default function(config, options) {
-  return safe(formatConfigHelper(config));
+export function safeFormatConfig(config, options) {
+  console.log(config);
+  return JSON.stringify(config);
 }
+
+export default Ember.Helper.helper(safeFormatConfig);
```

Comment:
Looks like [the tests failed](https://travis-ci.org/travis-ci/travis-web/jobs/144196014) because we forgot to `import Ember from 'ember';` at the top 😥